docs/docs/using-gatsby-image.md

```diff
@@ -81,15 +81,33 @@ module.exports = {
 
 4. Write a GraphQL query using one of the included [GraphQL “fragments”](/packages/gatsby-image/#fragments) which specify the fields needed by `gatsby-image` to create a responsive, optimized image. This example queries for an image at a path relative to the location specified in the `gatsby-source-filesystem` options using the `GatsbyImageSharpFluid` fragment.
 
-```graphql
-file(relativePath: { eq: "images/default.jpg" }) {
+```jsx:title=src/pages/my-dogs.js
+import React from "react"
+import { graphql } from "gatsby"
+import Layout from "../components/layout"
+
+export default ({ data }) => {
+  return (
+    <Layout>
+      <h1>I love my corgi!</h1>
+    </Layout>
+  )
+}
+
+// highlight-start
+export const query = graphql`
```

Comment:
The screencast embeds use the useStaticQuery hook. It seems like these should match if being expanded for more context.

Reply:
Updated :) Thanks!